gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js

```diff
@@ -905,7 +905,9 @@ GbGradeTable.renderTable = function (elementId, tableData) {
     const rowIndex = range._range?.end?.row;
   
     const table = GbGradeTable.instance;
-  
+
+    if (!table.getRows()[rowIndex]?.getCells()[colIndex]) return;
+
     const cellRect = table.getRows()[rowIndex].getCells()[colIndex].getElement().getBoundingClientRect();
```

Comment:
```suggestion
    const cell = table.getRows()[rowIndex]?.getCells()[colIndex];
    if (!cell) return;
    const cellRect = cell.getElement().getBoundingClientRect();
```

Reply:
It's less brittle if you just use the array lookups once, and less cpu in the browser.

Reply:
Thanks :)